### PR TITLE
add PodDisruptionBudget

### PR DESCRIPTION
**Pull request by D1abloRUS**
Signed-off-by: Sergey Monakhov <sergey.monakhov@gmail.com>

Added PodDisruptionBudget template to helm chart

### CODE REVIEW

**Comment by tenmozes** [comment]
will take a look this evening

File: charts/victoria-metrics-single/values.yaml [hunk 1]
```diff
@@ -11,10 +11,18 @@ rbac:
...
   extraLabels: {}
   # annotations: {}
 
+## See `kubectl explain poddisruptionbudget.spec` for more
+## ref: https://kubernetes.io/docs/tasks/run-application/configure-pdb/
+podDisruptionBudget:
+  enabled: false
+#  minAvailable: 1
+#  maxUnavailable: 1
+#  extraLabels: {}
```
Comment (tenmozes) [inline charts/victoria-metrics-single/values.yaml]: no need to comment extraLabels

File: charts/victoria-metrics-single/Chart.yaml [hunk 2]
```diff
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: 1.32.8
 description: Victoria Metrics Single version - high-performance, cost-effective and scalable TSDB, long-term remote storage for Prometheus
 name: victoria-metrics-single
-version: 0.0.1
+version: 0.1.0
```
Comment (tenmozes) [inline charts/victoria-metrics-single/Chart.yaml]: please bump to 0.0.2

File: charts/victoria-metrics-single/values.yaml (hunk 1 above)
Comment (D1abloRUS) [reply charts/victoria-metrics-single/values.yaml]: fixed

File: charts/victoria-metrics-single/Chart.yaml (hunk 2 above)
Comment (D1abloRUS) [reply charts/victoria-metrics-single/Chart.yaml]: fixed

**Comment by tenmozes** [comment]
perfect, thanks

**Pull request merged**